src/display/text_layer.js

```diff
@@ -133,7 +127,8 @@ const renderTextLayer = (function renderTextLayerClosure() {
     const textDivProperties = {
       angle: 0,
       canvasWidth: 0,
-      isWhitespace: false,
+      hasEOL: geom.hasEOL,
+      isWhitespace: geom.isSpace,
```

Comment:
Is the `isWhitespace` property still necessary?

I know that it's being used in `expandTextDivs` below, however I believe that with the other changes in this patch you'll now need to *also* expand the spaces for things to work as intended (based on the test results).
Hence the `isWhitespace` property, and *all* of its uses, should probably be removed now!?

Reply:
I checked with `tracemonkey.pdf` but it doesn't seem necessary to do it.

Reply:
Looking at the *latest* test results, e.g. the `tracemonkey-text-enhance` tests, it seems pretty clear that you'll actually need to expand spaces as well now (since the point is that the textLayer elements should fill the available area in `enhanced`-mode).

Hence I still believe that my previous comment, about simply removing `isWhitespace` and its remaining usage, still applies here.